src/negotiation/order.ts

```diff
@@ -152,3 +237,29 @@ export function fromNominal(
   }
   return new BigNumber(10).pow(decimals).times(nominalAmount);
 }
+
+export function orderParamsToTradingPair(orderParams: OrderParams): string {
```

Comment:
Why don’t you return an array or tuples or an object?

Reply:
This needs to be a primitive type such as string or integer so it can be used as key for the hasmap and in the url of the REST API of makernegotiator